+++
title = "Error-corrected deep targeted sequencing of circulating cell-free DNA from colorectal cancer patients for sensitive detection of circulating tumor DNA"
date = "2024-01-02"

[extra]
id = "GDK000009"
access_type = "Controlled"

[[extra.contacts]]
name = "Claus Lindbjerg Andersen"
email = "cla@clin.au.dk"

[[extra.contacts]]
name = "Mads Heilskov Rasmussen"
email = "mads.heilskov@clin.au.dk"
+++

# Study description

The study comprised a total of 107 healthy individuals and 382 patients with stage I-IV colorectal cancer (n=365) or colorectal adenomas (n=17) (collectively referred to as "CRC patients").


* Dataset A: Whole-exome sequencing of tumor DNA and paired normal DNA from peripheral blood mononuclear cells (PBMCs) from 345 patients with stage I-IV colorectal cancer (CRC).
* Dataset B: Whole-genome sequencing of tumor DNA and paired normal DNA from PBMCs from 17 patients with colorectal adenomas.
* Dataset C: Targeted sequencing (by Duplex sequencing\[1\]) of tumor DNA and paired normal DNA from PBMCs from 19 patients with stage I-IV colorectal cancer (CRC).
* Dataset D: Targeted sequencing (by UMIseq) of cfDNA from 381 pre-operative plasma samples from CRC patients.
* Dataset E: Targeted sequencing (by UMIseq) of cfDNA from 107 healthy individuals.
* Dataset F: Targeted sequencing (by UMIseq) of normal DNA from PBMCs from 382 CRC patients.
* Dataset G: Targeted sequencing (by UMIseq) of cfDNA from 26 post-operative plasma samples from CRC patients.


Dataset ID | Sample type                    | Sequencing panel        | Sequencing platform
-----------|--------------------------------|-------------------------|----------------------
A          | Tumor DNA and normal DNA       | Whole-exome sequencing  | Illumina NovaSeq 6000
B          | Tumor DNA and normal DNA       | Whole-genome sequencing | Illumina NovaSeq 6000
C          | Tumor DNA and normal DNA       | Targeted sequencing     | Illumina NovaSeq 6000
D          | cfDNA from CRC patients        | Targeted sequencing     | Illumina NovaSeq 6000
E          | cfDNA from healthy individuals | Targeted sequencing     | Illumina NovaSeq 6000
F          | Normal DNA                     | Targeted sequencing     | Illumina NovaSeq 6000
G          | cfDNA from CRC patients        | Targeted sequencing     | Illumina NovaSeq 6000

# Original publication

Frydendahl et al. *Error-corrected deep targeted sequencing of circulating cell-free DNA from colorectal cancer patients for sensitive detection of circulating tumor DNA*

# Data access

External researchers (academic or commercial) interested in analysing the colorectal cancer datasets will need to contact the Data Access Committee via email to cla@clin.au.dk. The Data Access Committee is formed of Claus Lindbjerg Andersen, Mads Heilskov Rasmussen, and Ole Halfdan Larsen (Department of Clinical Medicine, Aarhus University). Due to Danish Law, for the authors to be allowed to share the data (pseudo-anonymized) it will require prior approval from The Danish National Committee on Health Research Ethics (or similar) for the specific new research goal. The author (based in Denmark) must submit the application for ethical approval, with the external researcher(s) as named collaborator(s). In addition to ethical approval, a Collaboration Agreement and a Data Processing Agreement is required, both of which must be approved by the legal office of the institution of the author (data owner) and the legal office of the institution of the external researcher (data processor).

### References

1. Christensen MH, Drue SO, Rasmussen MH, Frydendahl A, Lyskj√¶r I, Demuth C, et al. DREAMS: deep read-level error model for sequencing data applied to low-frequency variant calling and circulating tumor DNA detection. Genome Biol. 2023;24(1):99.
